lib/cucumber/rails.rb

```diff
@@ -1,5 +1,8 @@
-env_caller = File.dirname(caller.detect { |f| f =~ /\/env\.rb:/ }) if caller.detect { |f| f =~ /\/env\.rb:/ }
-if env_caller
+called_from_env_rb = caller.detect { |f| f =~ /\/env\.rb:/ }
+
+if called_from_env_rb
+  env_caller = File.dirname(called_from_env_rb) if called_from_env_rb
```

Comment:
I like this clean-up. There's a small nit: This final `if called_from_env_rb` isn't needed.